commcare_connect/opportunity/tables.py

```diff
@@ -75,7 +75,8 @@ class Meta:
 
 
 class AggregateColumn(columns.Column):
-    pass
+    def render_footer(self, bound_column, table):
+        return sum(1 if bound_column.accessor.resolve(row) else 0 for row in table.data)
```

Comment:
Does this still only total what is visible? I think we will want to fix that before adding the total row back.